canonical-web-and-design/ubuntu.com
# Update recaptcha on the /legal/contributors/agreement form

## Done

- Form assembly updated the recaptcha and I had to update the /legal/contributors/agreement form
- There was also an issue with the email the form was sending from, so I updated that on form assembly ([see this RT](https://rt.admin.canonical.com/Ticket/Display.html?id=132756))

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/legal/contributors/agreement
- See that the form isn't horrible looking (it needs a lot of css and classes from FA)
- See that the form submits in the inspector networking tab
